docs/azure.rst

```diff
@@ -274,4 +274,7 @@ azure.batch.pools.<name>.autoScale              Enable autoscaling feature for t
 azure.batch.pools.<name>.scaleFormula           Specify the scale formula for the pool identified with ``<name>``. See Azure Batch `scaling documentation <https://docs.microsoft.com/en-us/azure/batch/batch-automatic-scaling>`_ for details.
 azure.batch.pools.<name>.scaleInterval          Specify the interval at which to automatically adjust the Pool size according to the autoscale formula. The minimum and maximum value are 5 minutes and 168 hours respectively (default: `10 mins`)
 azure.batch.pools.<name>.schedulePolicy         Specify the scheduling policy for the pool identified with ``<name>``. It can be either ``spread`` or ``pack`` (default: ``spread``).
+azure.batch.pools.<name>.registry               Specify the container registry from which to pull the Docker images.
```

Comment:
Is really needed to have a different registry for each pool? it looks a bit overwhelming, I guess the most common scenario to have all images in the same registry for all pools

Reply:
I mirrored what the Batch API offers (actually they let you configure multiple registries for each pool). But I agree with you, it's overcomplicating the config. I moved the registry to the first level of the batch section.